benchmark/inclusion/inclusion.cc

```diff
@@ -513,13 +513,16 @@ namespace aspect
                                          VectorTools::L2_norm,
                                          &comp_p);
 
+      unsigned int n = this->get_solution().block(0).size() +
+                       this->get_solution().block(1).size();
```

Comment:
It would be good to use introspection.block_indices.pressure (and the same for the velocity) instead of just 0 and 1 here. Also, if the direct solver is used, pressure and velocity are in the same block, and then the numbers 0 and 1 are suddenly not correct any more.